build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi8/ubi:8.0
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.3
 
 RUN yum -y install net-tools
```

Comment:
It seems that this string causes build failure because ubi-minimal doesn't contain `yum` command. I've added net-tools (netstat) installation during debugging this https://infoblox.atlassian.net/browse/ATLAS-8402 issue. Since ATLAS-8402 issue has been fixed we can remove this line or change the base image to `ubi-minimal:8.3` which supports `yum`.

Reply:
Addressed in 3d9f55f